examples/06_manipulating_images/plot_affine_transformation.py

```diff
@@ -102,33 +102,29 @@
 from nilearn.image import resample_img
 
 img_in_mm_space = resample_img(
-    img, target_affine=np.eye(4), target_shape=(512, 512, 1), copy_header=True
+    img,
+    target_affine=np.eye(4),
+    target_shape=(512, 512, 1),
```

Comment:
Do we want this trailing comma to explode the line ?

Reply:
I think the multi-line is fine: if anything it may avoid some horizontal scrolling on smaller devices:

<img width="829" height="783" alt="image" src="https://github.com/user-attachments/assets/bd358e3e-5307-410b-9f80-1a512575bdd9" />

I think that if we wanted to be consistent in terms of mobile reading friendliness we'd enforce even shorter lines length in the doc but I think that'd be pushing things a bit far for no obvious gains (not sure how many people read the doc on their phone).

For now I can personally live with some inconsistencies #FML.